.travis.yml

```diff
@@ -1,14 +1,21 @@
 os:
-  - linux
-
+- linux
 language: go
-
 go:
-  - 1.9
-
+- 1.9
 services:
-  - docker
-
+- docker
 script:
-  - CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o certificate-controller .
-  - docker build -t certificate-controller:1.0 .
+- CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o certificate-controller
+  .
+- docker build -t easymile/certificate-controller:1.0-$(echo $TRAVIS_COMMIT | cut -c1-7) .
+- docker tag easymile/certificate-controller:1.0-$(echo $TRAVIS_COMMIT | cut -c1-7) easymile/certificate-controller:1.0-latest
```

Comment:
You repeat 4 times the Docker image repository. Use a variable to store it. The easiest way would be to take all those lines, put it in a script in the source tree and use it in the travis.yml. It will be cleaner that way.

Reply:
Or use a Makefile...

Reply:
Yep a Makefile is always handy.